src/Entity/IdeasWorkshop/Need.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityNameSlugTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *     name="ideas_workshop_need",
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="need_slug_unique", columns="slug")
+ *     }
+ * )
+ *
+ * @UniqueEntity("slug")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Need
+{
+    use EntityNameSlugTrait;
```

Comment:
in general, the slug value is used in URL to find a resource, do we really need to have a slug column for Need entity? will this entity be exposed with its URL (ex: `/needs/{slug}`)?

Reply:
Yes it will be exposed